shelter/racommand.go

```diff
@@ -20,12 +21,8 @@ EXAMPLE:
        # shelter mrenclave`,
 	Flags: []cli.Flag{
 		cli.StringFlag{
-			Name:  "ip",
-			Usage: "tcp socket ip to connect inclavared",
-		},
-		cli.StringFlag{
-			Name:  "port",
-			Usage: "tcp socket port to connect inclavared",
+			Name:  "addr",
+			Usage: "tcp or unix socket addr to connect inclavared such as '--addr=tcp://ip:port or --addr=unix://path'",
```

Comment:
change to `specify tcp or unix socket address, e.g, '--addr=tcp://ip:port or --addr=unix://path'`

Reply:
change done.